### ISSUE TITLE
[feature-request]: Add "notes" to channel subscription events

### ISSUE THREAD

**Issue by niteshbalusu11** [lightningnetwork/lnd#6848]
### Background

Today if there is a force closure for example, the only way I know of how to look at what caused the issue is by grepping the logs, it'll be nice the "channelEvents" API can have a "notes" key in its response body that can show notes of what caused the event like how its logged in the log file. I am working on some node monitoring stuff in DataDog and these notes can really help when monitoring important events.

### Your environment

* version of `lnd`: `0.15.0`
* which operating system (`uname -a` on *Nix) `ubuntu 22`
* version of `btcd`, `bitcoind`, or other backend: `bitcoind`
* any other relevant environment details

**Comment by ellemouton** [comment]
Thanks for the feature request @niteshbalusu11 :) 
Ok so you mean like "peer violated rule x and therefore we decided to force close with them" type of thing?

**Comment by niteshbalusu11** [comment]
> Thanks for the feature request @niteshbalusu11 :) Ok so you mean like "peer violated rule x and therefore we decided to force close with them" type of thing?

Right, whatever the reason was for the event to occur, like if an HTLC expired when a peer was offline, the peer used SCB, etc.
